service_upstart_linux.go

```diff
@@ -61,6 +63,10 @@ func (s *upstart) Platform() string {
 // Upstart will be replaced by systemd in most cases anyway.
 var errNoUserServiceUpstart = errors.New("User services are not supported on Upstart.")
 
+func (s *upstart) ConfigPath() (string, error) {
```

Comment:
As I previously mentioned, don't just wrap an existing private method, rename it to export it.

Reply:
Done